carapace-server/src/main/java/org/carapaceproxy/api/ConfigResource.java

```diff
@@ -47,10 +54,40 @@
 public class ConfigResource {
 
     private static final Logger LOG = Logger.getLogger(ConfigResource.class.getName());
+    private static final String CONFIGURATION_DUMPING_FILENAME = "server.dynamic.config";
 
     @javax.ws.rs.core.Context
     ServletContext context;
 
+    @GET
+    public Response dump() {
+        StreamingOutput configStream = (output) -> {
+            try {
+                HttpProxyServer server = (HttpProxyServer) context.getAttribute("server");
+                ConfigurationStore configStore = server.getDynamicConfigurationStore();
+                String lineSeparator = configStore.getProperty("line.separator", "\n");
```

Comment:
this shouldn't be needed
we can use LF (unix style)

another approach is to use the native platform line separator (System.getProperty("line.separator")) 
https://docs.oracle.com/javase/tutorial/essential/environment/sysprop.html